provider/aws/aws.go

```diff
@@ -144,7 +144,8 @@ func (p *AWSProvider) cloudformation() *cloudformation.CloudFormation {
 }
 
 func (p *AWSProvider) cloudwatch() *cloudwatch.CloudWatch {
-	return cloudwatch.New(session.New(), p.config())
+	config := p.config().WithLogLevel(aws.LogOff)
```

Comment:
Wouldn't this need to be on the `cloudwatchlogs` helper?

Can also be done without the extra variable.

Reply:
Oh interesting, I didn't even see the other client. Agree that that's probably what we want.